### ISSUE TITLE
"MULTI & LRANG" BUG

### ISSUE THREAD

**Issue by liuguanglg** [openresty/redis2-nginx-module#25]
When using MULTI AND LRANG together will cause error "Redis server returned invalid response near pos 18 in "

conf like this:
"
redis2_query MULTI;
redis2_query  LRANGE  "dict" 0 0;                                                                                            
redis2_query EXEC;
"

redis 127.0.0.1:6379> EXEC
1) 1) "2"
   2) "1"
MAY BE" 1) 1)"  tigger the error info.

**Comment by agentzh** [comment]
@liuguanglg Right, `ngx_redis2` does not really support Redis transactions. You may consider using the [lua-resty-redis](https://github.com/openresty/lua-resty-redis) library with the [ngx_lua](https://github.com/openresty/lua-nginx-module) module instead.

Thank you for the report anyway :)

**Comment by liuguanglg** [comment]
Thanks a lot!
lua-resty-redis is a good choice.
